components/Footer.js

```diff
@@ -89,15 +96,13 @@ export default function Footer (props) {
         <ul className='ft-legal-list'>
           <li>
             <NavLink
-              // FIXME: Broken link
-              href='/terms'>
+              href={isTermsMarkdown ? '/terms' : termsUrl}>
```

Comment:
Terms URL on the signup page needs fixing:

https://github.com/ibi-group/otp-admin-ui/blob/0a4b941c94b1d86cf22f4e67caf87453622a670b/components/ApiUserForm.js#L165

Reply:
It might also be worth including the privacy policy in that consent statement.

Reply:
Updated in https://github.com/ibi-group/otp-admin-ui/pull/36/commits/bb6ae2ca4e5042f67d1d91f90bfb1d5dc771907e.